### PR TITLE
Docs: Tags

### LINKED ISSUE #106: Documentation on how Tags work
### What's the Problem?

Tags are a very nice feature, however without documentation on how they are stored and clarifications regarding how they work when operations are done on files outside of Files, it can be too risky for power users to get invested into the system because they could risk losing the tag data or not being able to migrate/sync it to another system in the future.

### Solution/Idea

A documentation page that explains how Tags are stored, their limitations, what happens if files are modified outside of Files.
Ideally, a link to the documentation could be added to the UI of Files next to the settings menu for tags (maybe as an info button) in order to help with the discoverability of these details.

### Alternatives

Not sure what an alternative would be here

### Priorities

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to accomplish W | Must |
| This proposal will allow end users to accomplish X | Should |
| This proposal will allow developers to accomplish Y | Could |
| This proposal will allow end users to accomplish Z | Won't |

### Files Version

_No response_

### Windows Version

_No response_

### Comments

_No response_

### PR DESCRIPTION
**Pull request by yaira2**
Closes #106

### CODE REVIEW

**Comment by yaira2** [comment]
Please don't merge until the next update is released

File: src/routes/docs/configuring/tags/+page.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+title: Use tags to organize files on Windows
+---
+
+Tags are a way to organize your files and folders. You can assign one or more tags to any file or folder, and then use the tags to filter and sort your files. You can also use tags to search for your files.
+	
+## How to assign tags
+
+To assign a tag to a file or folder, follow these steps:
+
+- Right click the file or folder you want to tag.
```
Comment (chingucoding) [inline src/routes/docs/configuring/tags/+page.md:11]:
```suggestion
- Right-click the file or folder you want to tag.
```

File: src/routes/docs/configuring/tags/+page.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+You can also assign tags by dragging and dropping files or folders onto existing tags in the sidebar.
+
+## How to view and filter by tags
+
+To view and filter your files by tags, follow these steps:
+
+- You will see a list of your tags on the bottom of the left-hand sidebar
+- Tap on a tag to see all the files or folders with that tag.
+
+You can also click tags in the details layout to start a search. To do that, follow these steps:
```
Comment (chingucoding) [inline src/routes/docs/configuring/tags/+page.md:24]: Is it the "details layout" or rather "Details-Layout"? What do you think @yaira2 ?

File: src/routes/docs/configuring/tags/+page.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- Switch to the details layout.
+- There will be a tag column where you can see tags assigned to your files and folder
+- Tap on the tag you want to search for
+- You will see a list of all the files or folders with that tag.
+
+## How to edit or delete tags
+
+You can create and edit tags from the advanced page in the settings. To do that, follow these steps:
+
+- Tap on the "Settings" icon at the top right corner of the screen.
```
Comment (chingucoding) [inline src/routes/docs/configuring/tags/+page.md:35]:
```suggestion
- Tap on the "Settings" icon in the top right corner of the screen.
```

File: src/routes/docs/configuring/tags/+page.md (hunk 2 above)
Comment (yaira2) [reply src/routes/docs/configuring/tags/+page.md:24]: I usally refer to it as the way it is but I don't have a strong preference.

File: src/routes/docs/configuring/tags/+page.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- You will see a list of all the tags you have created.
+- You can tap on the "new tag" button to create a new tag, or the "edit" and "delete" buttons on existing tags to edit or delete them.
+- If you choose to edit, you can change the name or color of the tag.
+
+Note that deleting a tag will not delete the files or folders with that tag.
+
+
+## How do tags work behind the scenes?
+
+Instead of creating a database with a list of files and tags, we store the tag information in the file itself using alternate data streams. This allows users to move files around without losing their tags. Another benefit is that it decouples the feature from Files and allows other apps using the same system to support this feature.
```
Comment (chingucoding) [inline src/routes/docs/configuring/tags/+page.md:47]: Maybe break this line into multiple lines?

File: src/routes/docs/configuring/tags/+page.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+To view and filter your files by tags, follow these steps:
+
+- You will see a list of your tags on the bottom of the left-hand sidebar
+- Tap on a tag to see all the files or folders with that tag.
+
+You can also click tags in the details layout to start a search. To do that, follow these steps:
+
+- Switch to the details layout.
+- There will be a tag column where you can see tags assigned to your files and folder
```
Comment (chingucoding) [inline src/routes/docs/configuring/tags/+page.md:27]: I think merging this with the previous line would be better.

**Review by chingucoding — APPROVED** [review]

File: src/routes/docs/configuring/tags/+page.md (hunk 4 above)
Comment (yaira2) [reply src/routes/docs/configuring/tags/+page.md]: 👍

File: src/routes/docs/configuring/tags/+page.md (hunk 5 above)
Comment (yaira2) [reply src/routes/docs/configuring/tags/+page.md]: 👍

**Review by chingucoding — DISMISSED** [review]

**Pull request merged**